src/main/java/pocketpal/frontend/constants/MessageConstants.java

```diff
@@ -24,24 +24,29 @@ public final class MessageConstants {
             + "-d <description>" + NEWLINE
             + "-c <category>" + NEWLINE
             + "-p <price>" + NEWLINE
+            + "*All three options must be specified" + NEWLINE + NEWLINE
+            + "Available categories are:" + NEWLINE
+            + "CLOTHING, ENTERTAINMENT, FOOD," + NEWLINE
+            + "TRANSPORTATION, MEDICAL, PERSONAL," + NEWLINE
+            + "INCOME, UTILITIES, OTHERS" + NEWLINE + NEWLINE
```

Comment:
Perhaps we can add this in the main help message instead of under `/help add`? This would allow the users to see the categories available since they are used in `/edit` also. This will also address #198.